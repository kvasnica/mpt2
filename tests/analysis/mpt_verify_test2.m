function mpt_verify_test2

% tests mpt_verify() when input is a controller

load di_ctrl_N3_norm2

% feasible verification
X0=unitbox(2,1)+[3;0];
Xf=unitbox(2,0.1); 

f=mpt_verify(ctrl, X0, Xf, 100); 
mbg_asserttrue(f);


% infeasible verification
X0=unitbox(2,1)+[3;0];
Xf=unitbox(2,0.1)+[1; 0]; 

f=mpt_verify(ctrl, X0, Xf, 15); 
mbg_assertfalse(f);
