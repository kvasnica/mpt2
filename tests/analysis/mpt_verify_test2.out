Performing verification based on reachable sets...
Horizon 5	
Performing verification based on reachable sets...
Horizon 5	Horizon 10	Horizon 15	
